internal/common/labels.go

```diff
@@ -38,15 +39,20 @@ const (
 // Instance wide labels will be taken from the request if available
 func AddAppLabels(requestInstance *ssp.SSP, name string, component AppComponent, obj client.Object) client.Object {
 	labels := getOrCreateLabels(obj)
-	addInstanceLabels(requestInstance, labels)
-
-	labels[AppKubernetesNameLabel] = name
-	labels[AppKubernetesComponentLabel] = component.String()
-	labels[AppKubernetesManagedByLabel] = AppKubernetesManagedByValue
-
+	addCommonLabels(labels, requestInstance, name, component)
 	return obj
 }
 
+// AddAppLabels to the provided object metadata
+// Name will translate into the AppKubernetesNameLabel
+// Component will translate into the AppKubernetesComponentLabel
+// Instance wide labels will be taken from the request if available
+func AddAppLabelsToMeta(objectMeta *metav1.ObjectMeta, requestInstance *ssp.SSP, name string, component AppComponent) *metav1.ObjectMeta {
```

Comment:
This function is not needed, see my above comment.

Reply:
Done.